### DISCUSSION TITLE
Does the validation set is actually needed when training only one fold?

### DISCUSSION THREAD

**Discussion by KanielDatz** [MIC-DKFZ/nnUNet#2351] [Q&A]
Hello and thanks a lot for your work!
I'm not using the ensemble of 5 folds as you intended, I'm training only 1 fold and using only  a single model for my cause. 
Is there a use for the val set except for the estimation during training and the ensembling at the inference?
Do I need the validation set? maybe I better just use those extra frames for some more training data? might get better results on test.
If so , I should do it by editing the splits_final.json or is there another way intended?
